apps/crn-server/src/data-providers/research-outputs.data-provider.ts

```diff
@@ -173,6 +176,56 @@ export class ResearchOutputSquidexDataProvider
     };
   }
 
+  async fetchDrafts(options: {
```

Comment:
I think I mentioned that in my previous review but we aim to only use `create` `update`, `fetch` and `fetchById` in our providers. To be fair I don't think you need a separate method either here or in your controller but since it's quite a common practice in this repo to add such methods to controllers, at least make sure to reduce it in the data-provider. 

It's come up recently elsewhere, maybe we should create a generic `DataProvider` interface to enforce this?

Reply:
Will address this issue in a separate PR